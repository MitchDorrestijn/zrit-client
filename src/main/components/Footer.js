import React from 'react';

import '../css/footer.css';

//this stateless functional component renders the footer
const Footer = () => {
	return (
    <footer>
		  <small>Heeft u vragen? Bel ons:</small>
      <span>020 - 123 45 67</span>
    </footer>
	);
}

export default Footer;
