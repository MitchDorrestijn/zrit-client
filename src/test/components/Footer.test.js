import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../../main/components/Footer';

const wrapper = shallow(<Footer />);

test('<Footer /> should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('<Footer /> should have one type of <footer></footer> tags', () => {
  expect(wrapper.find('footer').length).toBe(1);
});

test('<Footer /> should have one type of <small></small> tags', () => {
  expect(wrapper.find('small').length).toBe(1);
});

test('<Footer /> should have one type of <span></span> tags', () => {
  expect(wrapper.find('span').length).toBe(1);
});

test('The phonenumber in the <Footer /> should match 020 - 123 45 67', () => {
  expect(wrapper.find('span').text()).toBe('020 - 123 45 67');
});
