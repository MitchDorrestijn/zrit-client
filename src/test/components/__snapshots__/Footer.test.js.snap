// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
<footer>
  <small>
    Heeft u vragen? Bel ons:
  </small>
  <span>
    020 - 123 45 67
  </span>
</footer>
`;
